### PR TITLE
docs(Table): sticky

### PR DESCRIPTION
**Pull request by vaynevayne**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [x] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Workflow
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |           |
| 🇨🇳 Chinese |           |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [ ] Doc is updated/provided or not needed
- [ ] Demo is updated/provided or not needed
- [ ] TypeScript definition is updated/provided or not needed
- [ ] Changelog is provided or not needed

---

### <samp>🤖 Generated by Copilot at 095bee1</samp>

Update Chinese documentation of `fixed-header` table component to warn about overflow:hidden issue. The change helps users avoid a potential problem that may affect the component's functionality.

### <samp>🤖 Generated by Copilot at 095bee1</samp>

*  Add a note to the Chinese documentation of the fixed-header table component to warn users about the possible issue of overflow:hidden on the outer container ([link](https://github.com/ant-design/ant-design/pull/43513/files?diff=unified&w=0#diff-6a8ba8cf2f7be7472dafbb1f2098b9b7aef84e3b87a4af3f5f8e91e8905b5f88L5-R5))

### CODE REVIEW

**Comment by li-jia-nan** [comment]
单词和汉字之间需要保留空格，另外英文文档也可以顺便补充一下

**Comment by li-jia-nan** [comment]
另外你这个单词好像拼错了

File: components/table/demo/fixed-header.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 方便一页内展示大量数据。
 
-需要指定 column 的 `width` 属性，否则列头和内容可能不对齐。如果指定 `width` 不生效或出现白色垂直空隙，请尝试建议留一列不设宽度以适应弹性布局，或者检查是否有[超长连续字段破坏布局](https://github.com/ant-design/ant-design/issues/13825#issuecomment-449889241)。
+需要指定 column 的 `width` 属性，否则列头和内容可能不对齐。如果指定 `width` 不生效或出现白色垂直空隙，请尝试建议留一列不设宽度以适应弹性布局，或者检查是否有[超长连续字段破坏布局](https://github.com/ant-design/ant-design/issues/13825#issuecomment-449889241), 此外需要保证table外层容器没有 overflow:fidden。
```
Comment (afc163) [inline components/table/demo/fixed-header.md:5]: 这句话很奇怪，最好有相关的重现演示

**Comment by vaynevayne** [comment]
> 另外你这个单词好像拼错了

haole

**Comment by li-jia-nan** [comment]
> haole

再看一下上面大佬的留言👆

**Comment by afc163** [comment]
楼主没反馈，看到后欢迎回复，先关了。

**Pull request closed without merging**